bakery/src/scripts/check_feed.py

```diff
@@ -25,13 +25,17 @@ def main():
     # file in S3 bucket. If the book is not complete, check pending and retry
     # states to see whether or not it has errored or timed out too many times
     # to be queued again.
+
     for book in feed_data:
         # Check for loop exit condition
         if books_queued >= max_books_per_run:
             break
 
+        book_identifier = book['slug'] if book.get(
+            'slug') else book['collection_id']
```

Comment:
```suggestion
        book_identifier = (book['slug']
                           if 'slug' in book
                           else book['collection_id'])
```
or
```suggestion
        book.get('slug',book.get('collection_id'))
```
might be a little more idiomatic.